src/tests/ftest/datamover/negative.py

```diff
@@ -61,10 +64,11 @@ def test_dm_bad_params_dcp(self):
         pool1 = self.create_pool()
 
         # Create a special container to hold UNS entries
-        uns_cont = self.create_cont(pool1)
+        uns_cont = self.get_container(pool1)
 
         # Create a test container
-        cont1 = self.create_cont(pool1, True, pool1, uns_cont)
+        cont1_path = join(self.dfuse.mount_dir.value, pool1.uuid, uns_cont.uuid, 'uns1')
+        cont1 = self.get_container(pool1, path=cont1_path)
```

Comment:
(pylint-unexpected-keyword-arg) Unexpected keyword argument 'path' in method call